From: intrigeri <intrigeri@riseup.net>
Date: Wed, 19 Feb 2025 11:02:34 +0000
Subject: aa-notify: point to Debian documentation

Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/904436
---
 utils/notify.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/notify.conf b/utils/notify.conf
index 1524a45..e428fa8 100644
--- a/utils/notify.conf
+++ b/utils/notify.conf
@@ -31,7 +31,7 @@ ignore_denied_capability="sudo,su"
 # message_body="This is a custom notification message."
 
 # OPTIONAL - custom notification message footer
-# message_footer="For more information visit https://foo.com"
+message_footer="For more information visit https://wiki.debian.org/AppArmor"
 
 # OPTIONAL - custom notification filtering
 # filter.profile=""
